src/backtrace/dbghelp64.rs

```diff
@@ -114,6 +115,8 @@ pub unsafe fn trace(cb: &mut dyn FnMut(&super::Frame) -> bool) {
             break;
```

Comment:
As this code implements [backtrace::trace](https://docs.rs/backtrace/latest/backtrace/fn.trace.html), this `break` fulfills our caller's contract and we must take it before looping again. A comment should be left to this effect.

Reply:
The reason I say "before looping again" is that it's not immediately obvious to me exactly whether the `break` should happen before or after `RtlVirtualUnwind` is called again.

Reply:
I've added a comment here and for each of the `break` statements to explain why we are exiting the loop.